packages/api/package.json

```diff
@@ -74,6 +74,7 @@
     "@lodestar/params": "^1.4.3",
     "@lodestar/types": "^1.4.3",
     "@lodestar/utils": "^1.4.3",
+    "@lodestar/fork-choice": "1.4.3",
```

Comment:
can you avoid this?
Maybe use ProtoNodeSszType instead?

Reply:
thanks @wemeetagain , that works well 👍